src/key-derivation/hashing-password.md

```diff
@@ -0,0 +1,248 @@
+# Password Hashing
+
+RustCrypto provides a number of [password hash functions], which takes arbitrary data (a password)
+and produces a hash with parameters, which is typically stored in a database. When an unknown
```

Comment:
"with parameters" is part of the function, no?  Not part of the output.

Reply:
That is correct, I reworded to hopefully make this more clear